### DISCUSSION TITLE
有没有通过文件的URL地址获取文件内容，然后进行编辑保存的功能？

### DISCUSSION THREAD

**Discussion by xxxx96923** [dream-num/univer#2028] [Q&A | 常见问题]
我看着好像都是数据库获取内容

**Comment by Dushusir** [comment]
从文件 URL 地址获取内容，本质上是解析文件内容做导入，可以参考这个 XLSX 导入导出 Demo https://univer.ai/pro/examples/sheets-exchange 。源码可参考 https://github.com/dream-num/univer.ai/tree/main/pro/examples/sheets-exchange .

编辑保存可以通过 Facade API 本地保存表格数据的方式，或者使用我们的协同方案。

**Reply by zhang-ke-ke** [reply]
这个需要部署后端服务，呜呜呜，求大佬把导入导出剥离出来

**Comment by yuluo-yx** [comment]
https://github.com/dream-num/univer.ai/tree/main/pro/examples/sheets-exchange . 这个源码地址 404 了
能重新给一个吗

**Reply by hexf00** [reply]
https://univer.ai/guides/sheet/features/import-export#the-importexport-api-is-not-functioning-properly-
